components/bl-pdfViewer-component/src/index.js

```diff
@@ -12,7 +12,7 @@ export default function PdfViewer({ component, eventHandlers }) {
   const [pageIndex, setPageIndex] = useState(1);
   const [documentRef, setDocumentRef] = useState();
   const [pageRef, setPageRef] = useState();
-  const [controlsDisabled, setControlsDisabled] = useState(false);
+  const [isControlDisplay, setIsControlDisplay] = useState(false);
```

Comment:
Maybe will be better to rename it
`isControlVisible` or `displayControl`

Also it's plural, isn't it?